source/particle/generator/interface.cc

```diff
@@ -61,6 +61,20 @@ namespace aspect
         // Try to find the cell of the given position. If the position is not
         // in the domain on the local process, throw a ExcParticlePointNotInDomain
         // exception.
+#if DEAL_II_VERSION_GTE(9,3,0)
+        std::pair<const typename parallel::distributed::Triangulation<dim>::active_cell_iterator,
+            Point<dim> > it =
+              GridTools::find_active_cell_around_point<> (this->get_mapping(), this->get_triangulation(), position);
```

Comment:
Is the cell guaranteed to be locally owned if valid?

Reply:
I do not know, that is why I kept the additional assert below. I guess the function could return a ghost cell.

Reply:
Sorry, I didn't scroll to the right in the assert.